src/Compilation.zig

```diff
@@ -3009,7 +3009,7 @@ fn addBuildingGLibCJobs(comp: *Compilation) !void {
     });
 }
 
-fn wantBuildLibCFromSource(comp: Compilation) bool {
+pub fn wantBuildLibCFromSource(comp: Compilation) bool {
```

Comment:
This and the other `wantBuild...` function no longer need to be made `pub`

Reply:
done